### DISCUSSION TITLE
Abstract title option

### DISCUSSION THREAD

**Discussion by eteitelbaum** [quarto-dev/quarto-cli#4095] [Feature Requests]
For scientific publishing, would it be possible to have an YAML header argument to add a simple abstract title that reads "Abstract" like you would see in a typical [journal article](https://www.science.org/doi/10.1126/science.ade8190)? Something like `abstract-title: true` and then options for aligning it like `abs-title-align: center`. I realize that you can do this by including LaTeX code for a .pdf document, but it might be more convenient as a YAML argument.

**Comment by cscheid** [comment]
Thanks for the suggestion!

Those kinds of format specifications change a lot from journal to journal. We currently lean towards encouraging people to use a custom format (like the ones you can see on https://github.com/quarto-journals) rather than attempting to have our baseline format fit all needs.

**Reply by eteitelbaum** [reply]
Thanks so much! And thanks for your work on Quarto. My first observation is that this list is rather short and leans towards journals in the hard sciences. So might be good to diversify on that front. But also, one thing that my colleagues, students and I use Quarto and R Markdown for is academic working papers for presentation at workshops, posting on personal websites, platforms like repec or researchgate, etc.. While working papers don't have formatting requirements there are some norms that people follow and an abstract title is one thing that you see a lot. So either it would be good to have some working paper templates or some more options for customizing stuff like this rather than have to either go full bore with a journal template or hack together an adequate solution.

**Reply by cscheid** [reply]
> My first observation is that this list is rather short and leans towards journals in the hard sciences. So might be good to diversify on that front.

These templates have only existed since quarto 1.2, which came out only a couple of months ago ;) so give the community some time and I'm sure it'll improve!

> So either it would be good to have some working paper templates or some more options for customizing stuff like this rather than have to either go full bore with a journal template or hack together an adequate solution.

Would you like to contribute a template for working papers? I would be willing to put in a couple of one-on-one hours with you on Zoom to help you with it.

**Reply by eteitelbaum** [reply]
Thanks. Yes, I would be very interested in learning how to create templates! How can we coordinate? 

I did find one (see my comment below) that works pretty well for what I do, but would still be interested in tweaking it and working on templates for some political science and development studies journals.

**Reply by cscheid** [reply]
We have a starter template repo for new formats here, which would be a good starting point for you to fork from and play around a bit: https://github.com/quarto-journals/article-format-template

But also, reach me at carlos.scheidegger@posit.co.

**Reply by eteitelbaum** [reply]
Will do. Thanks!

**Comment by eteitelbaum** [comment]
Looking around a bit more, I found [this group of templates](https://github.com/andrewheiss/hikmah-academic-quarto)  that was linked to the Quarto site along with some others on a page about [custom formats](https://quarto.org/docs/extensions/listing-formats.html). There is an working paper template there that supports `biblatex-chicago` and has the issues with abstract formatting (and many others) all sorted out. This is exactly what I needed !
